raphtory-graphql/src/model/graph/graph.rs

```diff
@@ -59,15 +59,37 @@ impl GqlGraph {
         let name = self.name.clone();
         GqlGraph::new(name, self.graph.valid_layers(names))
     }
+
+    async fn exclude_layers(&self, names: Vec<String>) -> GqlGraph {
+        let name = self.name.clone();
+        GqlGraph::new(name, self.graph.exclude_layers(names).unwrap())
+    }
+
     async fn layer(&self, name: String) -> GqlGraph {
         GqlGraph::new(self.name.clone(), self.graph.valid_layers(name))
     }
 
+    async fn exclude_layer(&self, name: String) -> GqlGraph {
+        GqlGraph::new(self.name.clone(), self.graph.exclude_layers(name).unwrap())
```

Comment:
We need to make a valid_exclude_layers - one becuse it is useful and two because it is quite dangerous to have an unwrap here.